apm-sniffer/apm-sdk-plugin/kafka-plugin/src/main/java/org/apache/skywalking/apm/plugin/kafka/KafkaConsumerInterceptor.java

```diff
@@ -60,7 +60,7 @@ public Object afterMethod(EnhancedInstance objInst, Method method, Object[] allA
         //
         if (records.size() > 0) {
             ConsumerEnhanceRequiredInfo requiredInfo = (ConsumerEnhanceRequiredInfo)objInst.getSkyWalkingDynamicField();
-            AbstractSpan activeSpan = ContextManager.createEntrySpan(OPERATE_NAME_PREFIX + requiredInfo.getTopics() + "/" + requiredInfo.getGroupId() +  CONSUMER_OPERATE_NAME_SUFFIX, null).start(requiredInfo.getStartTime());
+            AbstractSpan activeSpan = ContextManager.createEntrySpan(OPERATE_NAME_PREFIX + requiredInfo.getTopics() + CONSUMER_OPERATE_NAME + "/" + requiredInfo.getGroupId(), null).start(requiredInfo.getStartTime());
```

Comment:
`/` append into `CONSUMER_OPERATE_NAME` to reduce a string operator.

Reply:
Okay